_includes/calendar.js

```diff
@@ -0,0 +1,93 @@
+$(document).ready(function() {
+  var adjustClasses, tooltipHide, tooltipShow, $tip, tip, $widget, now;
+  now = Date.now();
+  $widget = $("#calendar");
+
+  // Create tooltips on events
+  tip = '<div id="fc-tooltip" class="fc-tooltip hidden"></div>';
+  $tip = $(tip).appendTo($widget).hide().removeClass('hidden');
+
+  // Display the tooltip during mouseover
+  tooltipShow = function(event, jsEvent, view) {
+    var $item, eventInfo, eventText, label, value;
+    $item = $(this);
+
+    // Collect the event info
+    eventInfo = {
+      Title:    event.title,
+      Host:     event.speaker,
+      Location: event.location,
+      Time:     !event.allDay ? event.start.format('h:mm a z') : void 0,
+      Topic:    event.tags ? event.tags.join() : void 0
+    };
+
+    // Format the event text for the tooltip
+    eventText = (function() {
+      var results;
+      results = [];
+      for (label in eventInfo) {
+        value = eventInfo[label];
+        if (value) {
+          results.push("<div class=\"fc-tip-" + (label.toLowerCase()) + "\">\n  <span class=\"label\">" + label + ":</span>\n  " + value + "\n</div>");
+        }
+      }
+      return results;
+    })();
+
+    // Position, inject tooltip text, and fade tooltip in
+    $tip.offset({
+      top: $item.offset()["top"] - 20,
+      left: $item.offset()["left"] - $tip.width() - 40
+    }).html(eventText).stop().fadeTo(200, 1);
+  };
+
+  // Hide the tooltip
+  tooltipHide = function(event, jsEvent, view) {
```

Comment:
Why not making all these functions outside of the `.ready` scope? I don't see any reason to define them inline and save them in variables